.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--- Prepend PR title with [WIP] if work in progress. Remove when ready for review. -->
+
+<!--- For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments -->
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Documentation Update
+
+## Description
+
+## Related Tickets & Documents
+
+## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
```

Comment:
Could we replace this heading with `Before & After`?

That implies screenshots. Netlify should also be able to give us a preview before deploy, so am not too concerned about asking for recordings. :) GitHub also has a limit of 25mb fo files and 10mb for images (like gifs).